.clang-format

```diff
@@ -0,0 +1,219 @@
+---
+
+# Documentation on clang-format options here:
+# https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+
+Language: Cpp
+Standard: c++20
+
+# Google style is a good starting point; modifiers attempt to match style
+# guide here: https://github.com/ceph/ceph/blob/main/CodingStyle
+BasedOnStyle: Google
+IndentWidth: 2
+TabWidth: 8
+UseTab: Always
+MaxEmptyLinesToKeep: 2
+ContinuationIndentWidth: 4
+
+# The intended impact here is to allow up to 96 characters, but break at 80 if
+# 96 is exceeded.
```

Comment:
96?

Reply:
This works with the penalty values to allow some lines > 80 if it makes sense